test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs

```diff
@@ -1,5 +1,5 @@
-{{!-- The following component displays Ember's default welcome message. --}}
+{{! The following component displays Ember's default welcome message. }}
```

Comment:
Why are the `--` removed here? The original (assuming that `test/fixtures/codemod/local/my-app/app/templates/application.hbs` represents the input) uses `{{!--`, so I would expect the output to continue to use them.

Reply:
These were remnants from when the codemod did a lot more formatting, and the expected was altered to match. This has been reverted now.